apps/ios/SimpleX SE/ShareModel.swift

```diff
@@ -99,6 +99,8 @@ class ShareModel: ObservableObject {
             )
             try apiSetEncryptLocalFiles(privacyEncryptLocalFilesGroupDefault.get())
             let isRunning = try apiStartChat()
+            // TODO: Avoid duplicate event loop observer
+            // Here we should check, if the chat is already running and only start the polling `recvSimpleXMsg` if `isRunning == false`
```

Comment:
my suggestion is to not rely on isRunning response, and have an internal tracking that this loop already started or not.